test/e2e/apps/statefulset.go

```diff
@@ -1347,6 +1348,92 @@ var _ = SIGDescribe("StatefulSet", func() {
 			framework.ExpectNoError(err)
 		})
 	})
+
+	ginkgo.It("PVC should be recreated when pod is pending due to missing PVC [Feature:AutomaticPVCRecreation]", func() {
```

Comment:
I was wondering if I can add a tag [LocalVolumesRequired]? I did not see it in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests